Resources/skeleton/crud/actions/delete.php.twig

```diff
@@ -10,7 +10,7 @@
     public function deleteAction({{ entity }} ${{ entity|lower }}, Request $request)
     {
         $form = $this->createDeleteForm(${{ entity|lower }}->getId(), '{{ route_name_prefix }}_delete');
-        if ($form->handleRequest($request)->isValid()) {
+        if($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
```

Comment:
Missing space between "if" and parenthesis. Please run php-cs-fixer on your code before committing

Reply:
when I run "php-cs-fixer fix" on the delete.php.twig file no changes are made, presumably because it applies twig file rules, not php rules.

Is there some way to configure php-cs-fixer so that it applies php rules on the contents of a twig file (without choking on the twig code)?